lib/src/session.dart

```diff
@@ -551,23 +551,32 @@ enum SyncSessionErrorCode {
 /// These errors will be reported via the error handlers of the affected sessions.
 enum SyncResolveErrorCode {
   /// Host not found (authoritative).
-  hostNotFound,
+  hostNotFound(1),
```

Comment:
Previously you counted from 0. Was that a bug? Or is this?

Reply:
previously it was a bug

Reply:
see `realm_sync_error_resolve` in realm.h